src/minimal-webapi-openidconnect-v2/README.md

```diff
@@ -0,0 +1,121 @@
+
+:link: For more information about how to proctect your projects, please let's take a look at https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code. To know more about how this sample has been generated, please visit https://docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio-code
+
+## Prerequisites
+
+1. [Download .NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Register the web API application in your Azure Active Directory
+
+1. Choose a password for the Azure AD App
+
+   ```bash
+   AZURE_AD_APP_SECRET=<at-least-sixteen-characters-here>
+   ```
+
+1. Register a new Azure AD App
+
+   ```bash
+   AZURE_AD_APP_DETAILS_MINIMAL_API=$(az ad app create --display-name "active-directory-dotnet-minimal-api-aspnetcore" --password ${AZURE_AD_APP_SECRET} --available-to-other-tenants -o json) && \
+   AZURE_AD_APP_CLIENT_ID_MINIMAL_API=$(echo $AZURE_AD_APP_DETAILS_MINIMAL_API | jq ".appId" -r)
+   ```
+
+1. Disable the default scope for `user_impersonation`
+
+   ```bash
+   AZURE_AD_APP_USER_IMPERSONATION_SCOPE=$(echo $AZURE_AD_APP_DETAILS_MINIMAL_API | jq '.oauth2Permissions[0].isEnabled = false' | jq -r '.oauth2Permissions') && \
+   az ad app update --id $AZURE_AD_APP_CLIENT_ID_MINIMAL_API --set oauth2Permissions="$AZURE_AD_APP_USER_IMPERSONATION_SCOPE"
+   ```
+
+1. Create a new manifest scope for `access_as_user`
+
+   ```bash
+   cat > access_as_user_scope.json <<EOF
+   [
+     {
+       "adminConsentDescription": "Allows the app to access Minimal Api (active-directory-dotnet-minimal-api-aspnetcore) as the signed-in user.",
+       "adminConsentDisplayName": "Access Minimal Api (active-directory-dotnet-minimal-api-aspnetcore)",
+       "id": "1658e205-0e89-43a3-b107-b06a3e6dc60d",
+       "isEnabled": true,
+       "lang": null,
+       "origin": "Application",
+       "type": "User",
+       "userConsentDescription": "Allow the application to access Minimal (active-directory-dotnet-minimal-aspnetcore) on your behalf.",
+       "userConsentDisplayName": "Access Minimal Api (active-directory-dotnet-minimal-aspnetcore)",
+       "value": "access_as_user"
+     }
+   ]
+   EOF
+   ```
+
+1. Set the api uri and the `access_as_user` scope
+
+   ```bash
+   az ad app update --id $AZURE_AD_APP_CLIENT_ID_MINIMAL_API --identifier-uris "api://${AZURE_AD_APP_CLIENT_ID_MINIMAL_API}" --set oauth2Permissions=@access_as_user_scope.json
+   ```
+
+## Scaffold the web API by using the ASP.NET Core Minimal Api project template
+
+1. execute the following command to create the new web api project
+
+   ```bash
+   dotnet new webapi -minimal -o <name>
+   ```
+
+1. Add donet package for ASP.NET Core Identity
+
+   ```bash
+   dotnet add package Microsoft.Identity.Web
+   ```
+
+## Configure the web API
+
+1. create the `appsettings.json` file with the Azure AD app comfiguration
+
+   ```bash
+   cat > appsettings.json <<EOF
```

Comment:
add hardcoded file in the repo for content dev team to reference

Reply:
done | addressed from [d7e67](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/d7e67fb6e06da27118919420e28aae6a8147e1b4)

Reply:
Just curious, why would we still completely overwrite this file on CLI after it already exists in the solution?  If we only need to update the two lines, would it be better as a `sed` to update it vs overwriting the whole thing?

Reply:
I think it is fine either way, since the outcome would be exactly the same. More importantly, instructions presented in here are mainly for internal usage purposes, and also to give some ideas to our content dev team. Regarding the latter, I think current option can picture broader possibilities (inspire) rather than `sed`.

That said, any desire to replace current with `sed`, Im totally open for that provided that is the preference.

Reply:
fair enough, thanks for the explanation